explorer/.storybook/preview.js

```diff
@@ -6,4 +10,16 @@ export const parameters = {
       date: /Date$/,
     },
   },
-}
\ No newline at end of file
+}
+
+const withThemeProvider = (Story, context) => {
+  return (
+    <Emotion10ThemeProvider theme={theme}>
```

Comment:
Hey @BruceRodrigues just wondering for my own learning what the reason is for having both theme providers?

Reply:
This is to ensure that I'll have the same theme object using `@emontion/style` or any Material component